app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -127,7 +127,15 @@ export const AdditionalMarketData: React.FC<Props> = props => {
           <IconArbitrator size={'24'} />
           <AdditionalMarketDataSectionTitle>{arbitrator.name}</AdditionalMarketDataSectionTitle>
         </AdditionalMarketDataSectionWrapper>
-        <AdditionalMarketDataSectionWrapper isError={!verified}>
+        <AdditionalMarketDataSectionWrapper
+          data-arrow-color="transparent"
+          data-tip={
+            verified
+              ? 'This Market is verified by DXdao and therefore valid.'
+              : 'This Market is has not been verified and may be invalid.'
```

Comment:
"This Market has not been verified and may be invalid"
Please remove "is"
@pimato , what is your opinion?